### ISSUE TITLE
Error while building with docker

### ISSUE THREAD

**Issue by sandoche** [blockchain-certificates/cert-viewer#72]
I get the following error when I run `docker-compose build`

```
  Downloading https://files.pythonhosted.org/packages/5d/2e/be1693c4836d5fd6bc8bb322dd7eab41d5b7f6a0d59566449de321692244/cert-verifier-2.0.12.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-rw13f6cs/cert-verifier/setup.py", line 4, in <module>
        from pip.req import parse_requirements
    ModuleNotFoundError: No module named 'pip.req'
    
    ----------------------------------------

Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-rw13f6cs/cert-verifier/
```

**Comment by kimdhamilton** [comment]
There was a problem that dependencies in requirements.txt were out of date; fixed with 23bc1d757f2d4ae253528241631379caa4f2c7ba. Can you retry?

**Comment by sandoche** [comment]
It works thanks!
